### DISCUSSION TITLE
How to avoid using implicit string operator when Verifier is serializing the object?

### DISCUSSION THREAD

**Discussion by Iceeman76** [VerifyTests/Verify#980] [Q&A]
I have a class with an implicit operator like this:

`public static implicit operator string(Person person) => person.FullName;`

When I use Verify it will use the implicit operator instead of serializing the object.
Are there any settings or workaround for this?

**Comment by SimonCropp** [comment]
not that i know of. did you find any?

but why do you have an implicit  string conversion from person to FullName? isnt that prone to causing bugs?

**Reply by Iceeman76** [reply]
That was just an example, the main question is how to make Verifier avoid the operator?
Is it possible to provide a custom serializer or something?

**Reply by SimonCropp** [reply]
can you share a pull request with the scenario you expect
